app/views/courses/show.html.haml

```diff
@@ -13,10 +13,10 @@
           %p
             %b= semester
             \:
-          - instances.each do |instance|
+          - instances.sort_by { |instance| instance.section }.each do |instance|
             Section #{instance.section} taught by #{instance.professor.name}
             %br
-            = instance.meetings.first.schedule
+            = instance.meetings.try(:first).try(:schedule) || 'TBA'
```

Comment:
Yeah. Probably want this as an instance method on the course instance model. Also, this doesn't address the course instances with multiple meetings (like freshman sages). So something like:

```ruby
class CourseInstance
  def schedule
    return 'TBA' if (meetings || []).empty?
    meetings.map(&:schedule).join('; ')
  end
end
```